### PR TITLE
Update Lisk Core docker image settings

### PR DESCRIPTION
**Pull request by fchavant**
### What was the problem?

> We decided to disable all plugins by default, e.g.
>
> - lisk-framework-http-api-plugin
> - lisk-framework-forger-plugin
> - lisk-framework-monitor-plugin
> - lisk-framework-report-misbehavior-plugin
>
> We need to document how users can enable said plugins in the docker image README.

### How was it solved?

Remove `docker-compose.*.yml` files, update `README.md`.

### How was it tested?

Following the `README.md`'s instructions.

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -1,41 +1,7 @@
...
-	docker-compose down --volumes --remove-orphans
-
+.PHONY: build
 
 ifeq ($(CORE_VERSION),)
 CORE_VERSION := $(shell jq --raw-output .version ../package.json )
 endif
 build:
-	docker build --build-arg REGISTRY_URL=https://npm.lisk.io --build-arg NODEJS_VERSION=$(shell cat ../.nvmrc ) --build-arg CORE_VERSION=$(CORE_VERSION) -t=lisk/core:$(CORE_VERSION) .
+	podman build --build-arg REGISTRY_URL=https://npm.lisk.io --build-arg NODEJS_VERSION=$(shell cat ../.nvmrc ) --build-arg CORE_VERSION=$(CORE_VERSION) -t=lisk/core:$(CORE_VERSION) .
```
Comment (Nazgolze) [inline docker/Makefile:7]: Was this one intentional?

Comment (fchavant) [reply docker/Makefile:7]: Nope, good catch.

**Review by Nazgolze — CHANGES_REQUESTED** [review]
Dockerfile is still trying to pull in coldstart.sh

**Review by Nazgolze — CHANGES_REQUESTED** [review]
Aside from the one suggested change, there have been some tabs introduced into the README. And with that, I think I've hit the bottom of the barrel and check everything :+1:

**Review by Nazgolze — APPROVED** [review]

**Comment by shuse2** [comment]
@fchavant can you branch off from `release/3.0.0` so that we can include this in the testnet release?

File: docker/README.md [hunk 2]
```diff
@@ -1,79 +1,53 @@
-# Using docker-compose
+# Using the lisk/core docker image
 
-## Copy sample .env file
+lisk/core version 3 does not have any external dependencies and thus does not require using `docker-compose`.
 
-Choose one of the sample `.env.*` file and copy it to `.env`.
-You should set `ENV_LISK_VERSION` to the version of Lisk and can change the database configuration.
-Note: to configure Lisk Core itself, see below.
+Note: podman can be used. Simply replace occurences of `docker` with `podman` or `alias docker=podman`.
```
Comment (ManuGowda) [inline docker/README.md:5]: maybe link to podman? if someone doesn't know what it is or wants to install before replacing the occurrences?

Comment (ManuGowda) [inline docker/README.md:5]:
```suggestion
Note: podman can be used. Simply replace occurrences of `docker` with `podman` or `alias docker=podman`.
```

**Review by ManuGowda — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**